### ISSUE TITLE
Drop the year from the repo name?

### ISSUE THREAD

**Issue by trickeydan** [srobo/competitor-booklet-2019#1]
I think we could probably store the booklets for multiple years in here, including the cherry-picking of various previous booklets.

**Comment by RealOrangeOne** [comment]
I agree. It's very rare we need the year in the repo name. I see 2 solutions for this:

- Use `git` tags to tag the version used for a given year (as done in github.com/srobo/rules)
- Use subdirectories (as done in https://github.com/srobo/comp-floorplans)

**Comment by trickeydan** [comment]
I'm in favour of directories. It's easier for those who struggle with git.
